bundles/org.openhab.binding.network/src/main/feature/feature.xml

```diff
@@ -5,6 +5,9 @@
     <feature name="openhab-binding-network" description="Network Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
         <feature>openhab-core-model-script</feature>
+        <bundle dependency="true">mvn:fr.bmartel/jspeedtest/1.32.1</bundle>
```

Comment:
```suggestion
        <bundle prerequisite="true">wrap</bundle>
        <bundle dependency="true">wrap:mvn:fr.bmartel/jspeedtest/1.32.1</bundle>
        <bundle dependency="true">wrap:mvn:fr.bmartel/http-endec/1.04</bundle>
```

Reply:
Also please add proper descriptions to both bundles. (see other wrapped bundles)